src/main/webapp/trips/tripScript.js

```diff
@@ -23,7 +23,23 @@ $(document).ready(() => {
   setContentWidth('800px');
 });
 
-// Set the width of the content container.
+// Returns Promise with the sign in status in a boolean.
+function isSignedIn() {
+  return new Promise((resolve, reject) => {
+    fetch('/auth').then(response => response.json()).then((signInStatus) => {
+      resolve(signInStatus);
+    }).catch((error) => {
+      reject(error);
+    });
+  });
+}
+
+function displayHeader() {
+  // Display header for site.
+  document.getElementById('header').style.display = 'block';
+}
+
```

Comment:
I'm assuming these lines were added during some sort of merge issue, as I think this was the code I wrote; it isn't necessary any more, so you can remove it. (You can tell because it isn't called anywhere.)